tests/unit/amo/api/test_users.js

```diff
@@ -134,4 +135,33 @@ describe(__filename, () => {
       mockApi.verify();
     });
   });
+
+  describe('deleteUserPicture', () => {
+    const mockResponse = () => createApiResponse({
+      jsonData: createUserAccountResponse(),
+    });
+
+    const getParams = (params = {}) => {
```

Comment:
I think I've brought this up before, but it seems like we're doing a bunch of extra stuff here having a `getParams` function, which can even take an extra set of `params` to override the defaults, but then all we ever do with it is just call `getParams()`. I know it's how we usually do it, but I'm starting to question the value of all that extra code when we never really use it, and could just create the params inline. This is the only test that even has access to `getParams`, and it seems unlikely we'll add any others.

Reply:
We could indeed simplify a bit here.